OptimizelySDK.NetStandard16/OptimizelySDK.NetStandard16.csproj

```diff
@@ -53,6 +54,7 @@
         <Compile Include="..\OptimizelySDK\ProjectConfig.cs" />
         <Compile Include="..\OptimizelySDK\Utils\EventTagUtils.cs" />
         <Compile Include="..\OptimizelySDK\Utils\Validator.cs" />
+        <Compile Include="..\OptimizelySDK\Properties\AssemblyInfo.cs" />
```

Comment:
Can you please add one more assemblyinfo file & refer to that file. The problem is AssemblyInfo for .net should be named as OptimizelySDK.NetStandard16 while the referred info has OptimizelySDK name.